src/Moryx.Runtime.Kernel/KernelServiceCollectionExtensions.cs

```diff
@@ -40,7 +46,7 @@ public static void AddMoryxKernel(this IServiceCollection serviceCollection)
         public static void AddMoryxModules(this IServiceCollection serviceCollection)
         {           
             // Find all module types in the app domain
-            var loadedAssemblies = AppDomain.CurrentDomain.GetAssemblies();
```

Comment:
Are you sure about this? This casued issues in the past.

Reply:
`assembly.GetExportedTypes()` throws a System.NotSupportedException if called on a dynamic assembly. Without it will throw the exception, if that is desired as a safeguard/notification for developers then we need to discuss it. Otherwise it will just prevent the exception to be thrown (which was required for me to run the test)

Reply:
I am sorry, I misread the change. My mind switched old and new, so I thought you removed the change. Still slightly confused why there are already dynamic assemblies on system boot, but I'll accept it.